text/entities/0256-entities-data-model.md

```diff
@@ -0,0 +1,789 @@
+# Entities Data Model, Part 1
+
+This is a proposal of a data model to represent entities. The purpose of the data model
+is to have a common understanding of what an entity is, what data needs to be recorded,
+transferred, stored and interpreted by an entity observability system.
+
+<!-- toc -->
+
+- [Motivation](#motivation)
+- [Design Principles](#design-principles)
+- [Data Model](#data-model)
+  * [Minimally Sufficient Id](#minimally-sufficient-id)
+  * [Examples of Entities](#examples-of-entities)
+- [Entity Events](#entity-events)
+  * [EntityState Event](#entitystate-event)
+  * [EntityDelete Event](#entitydelete-event)
+- [Entity Identification](#entity-identification)
+  * [LID, GID and IDCONTEXT](#lid-gid-and-idcontext)
+  * [Semantic Conventions](#semantic-conventions)
+  * [Examples](#examples)
+    + [Process in a Host](#process-in-a-host)
+    + [Process in Kubernetes](#process-in-kubernetes)
+    + [Host in Cloud Account](#host-in-cloud-account)
+- [Prototypes](#prototypes)
+- [Prior Art](#prior-art)
+- [Alternatives](#alternatives)
+  * [Different ID Structure](#different-id-structure)
+  * [No Entity Events](#no-entity-events)
+  * [Merge Entity Events data into Resource](#merge-entity-events-data-into-resource)
+  * [Hierarchical ID Field](#hierarchical-id-field)
+- [Open questions](#open-questions)
+  * [Attribute Data Type](#attribute-data-type)
+  * [Classes of Entity Types](#classes-of-entity-types)
+  * [Multiple Observers](#multiple-observers)
+  * [Is Type part of Entity's identity?](#is-type-part-of-entitys-identity)
+  * [Choosing from Multiple Ids](#choosing-from-multiple-ids)
+- [Future Work](#future-work)
+- [References](#references)
+
+<!-- tocstop -->
+
+## Motivation
+
+This data model sets the foundation for adding entities to OpenTelemetry. The data model
+is largely borrowed from
+[the initial proposal](https://docs.google.com/document/d/1VUdBRInLEhO_0ABAoiLEssB1CQO_IcD5zDnaMEha42w/edit)
+that was accepted for entities SIG formation.
+
+This OTEP is step 1 in introducing the entities data model. Follow up OTEPs will add
```

Comment:
The OTEP does not introduce the _entire_ data model, which is important to note in some way. There will be "step 2 of data model". Otherwise it would be unclear why the OTEP is incomplete.